mod/node-api/handlers/proof/block_proposer.go

```diff
@@ -26,8 +26,9 @@ import (
 	"github.com/berachain/beacon-kit/mod/node-api/handlers/utils"
 )
 
-// GetBlockProposer returns the block proposer pubkey for the given block id
-// along with a merkle proof that can be verified against the beacon block root.
+// GetBlockProposer returns the block proposer pubkey for the given timestamp
+// id along with a merkle proof that can be verified against the beacon block
```

Comment:
nit: `for a give timestamp along`. Drop id

Reply:
@abi87 I would prefer using timestampID here since the beacon spec uses the blockID, stateID terminology. Similarly here timestampID conveys to the caller that they have to provide not just the timestamp but also prefix it with `t`. This function also then parses for timestampID with the `t` prefix, so providing just the timestamp would mistakenly return.